### DISCUSSION TITLE
FileUpload not show image from spatie/laravel-settings settings table

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#13084] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.71

### How can we help you?

<img width="760" alt="Screenshot 2024-06-04 at 12 28 46" src="https://github.com/filamentphp/filament/assets/121255432/01ccad1b-b23a-4505-98d6-5f32337274e9">

<img width="1000" alt="Screenshot 2024-06-04 at 12 29 03" src="https://github.com/filamentphp/filament/assets/121255432/98428c05-2da9-49e1-8c18-7dec12f1df38">

```
$this->migrator->add('general_settings.light_logo', filament()->getBrandLogo());

FileUpload::make('light_logo')
->directory('general-settings/logo')
->image()
->imageEditor(),
```

**Comment by leandrocfe** [comment]
The image should exist in the `general-settings/logo` because this uses the path, not url

**Reply by devhoussam1998** [reply]
```
like this? does not work

$this->migrator->add('general_settings.favicon', filament()->getFavicon());

->favicon(storage_path('app/public/general-settings/favicon-logo/favicon/favicon.ico'))

storage/
  app/
   public/
     general-settings/
       favicon-logo/
         favicon/
           favicon.ico
```

**Reply by leandrocfe** [reply, marked as answer]
yes, if you are using a public disk
